### ISSUE TITLE
Feature Request: Bookmarks Too?

### ISSUE THREAD

**Issue by patrickmac110** [TBM13/BrowserSearch#2]
Is there a way for you to enable it to search your bookmarks as well? Possibly also show what folder they're in?

**Comment by TBM13** [comment]
Yes, it should be possible. I actually had plans to do it (hence why I named this plugin `BrowserSearch` and not `HistorySearch`) but that's probably not happening anytime soon

**Comment by webysther** [comment]
Will be nice have this feature, @patrickmac110 you find any plugin capable of this?

**Comment by iizuraikami** [comment]
@webysther https://x.gd/UVVuk , https://x.gd/PrZW0 , https://x.gd/HmRUx

**Comment by iizuraikami** [comment]
In [PowerToys-Run-ChromeFavorite](https://github.com/octop162/PowerToys-Run-ChromeFavorite), edit line 14 of "FavoriteProvider.cs"edit line 14 to support other browsers
